Rakefile

```diff
@@ -8,3 +8,79 @@ Rake::TestTask.new do |t|
   t.test_files = FileList['test/**/*test.rb']
   t.verbose = true
 end
+
+def print(family, template, distro, major, minor, distro_prefix, release, validate, ipv4_subnet, ipv6_subnet)
```

Comment:
I think print is a built in function. Best not to override it.

Reply:
Hmmm Ruby and these aliases... changed.